ci-operator/step-registry/baremetalds/ofcir/teardown/baremetalds-ofcir-teardown-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds packet teardown command ************"
+
+set -x
+CIRFILE=$SHARED_DIR/cir
+if [ -e $CIRFILE ] ; then
+    OFCIRURL="https://ofcir-service.ofcir-system.svc.cluster.local/v1/ofcir"
```

Comment:
I don't remember if we discussed to move this into the vault or not. At least, it could help in removing the duplication between the steps